### PR TITLE
[StyleCop] Fix all the warnings in TextMatchers

### PR DESCRIPTION
**Pull request by Aliandi**
- Reorder properties, fields and methods
- Remove trailing whitespace
- Add trailing comma
- Reformat setters and getters

### CODE REVIEW

File: .NET/Microsoft.Recognizers.Text/Matcher/AcAutomaton.cs [hunk 1]
```diff
@@ -5,16 +5,15 @@ namespace Microsoft.Recognizers.Text.Matcher
 {
     public class AcAutomaton<T> : AbstractMatcher<T>
     {
-        protected readonly AaNode<T> root = new AaNode<T>();
```
Comment (JuanAr) [inline .NET/Microsoft.Recognizers.Text/Matcher/AcAutomaton.cs]: Extract the changes related to the field rename/promote to property into another PR, AND keep the other changes (reordering, spacing, new lines, etc.) in this PR.

**Review by JuanAr — CHANGES_REQUESTED** [review]
Split this PR into two: The non-breaking changes PR and the breaking changes PR.

File: .NET/Microsoft.Recognizers.Text/Matcher/AcAutomaton.cs (hunk 1 above)
Comment (Aliandi) [reply .NET/Microsoft.Recognizers.Text/Matcher/AcAutomaton.cs]: Extracted changes to new branch

Comment (JuanAr) [reply .NET/Microsoft.Recognizers.Text/Matcher/AcAutomaton.cs]: By "extract" I mean: Revert all changes related to the field rename/promote to property in this branch. and move them into another branch.

Comment (Aliandi) [reply .NET/Microsoft.Recognizers.Text/Matcher/AcAutomaton.cs]: Forgot to push, now its ok!

**Review by JuanAr — APPROVED** [review]
LGTM

**Pull request closed without merging**